cli/__tests__/logger.test.js

```diff
@@ -0,0 +1,60 @@
+const logger = require('../logger');
+
+describe('logger', () => {
+  beforeEach(() => {
+    global.console.log = jest.fn();
+    global.console.dir = jest.fn();
+  });
+
+  const simpleMethods = [
+    { method: 'log', expectedPrefix: '' },
+    { method: 'task', expectedPrefix: '👍  ' },
+    { method: 'start', expectedPrefix: '🔥  ' },
+    { method: 'end', expectedPrefix: '✅  ' },
+    { method: 'info', expectedPrefix: 'ℹ️  ' },
+  ];
+
+  const advancedMethods = [
```

Comment:
I think calling these simple and advanced isnt' really the distinction. It's about whether they get passed extra data yes? 

Maybe textCommands objectCommands or something like that